native/src/main/java/io/ballerina/stdlib/grpc/Message.java

```diff
@@ -1186,6 +1300,74 @@ private int computeMessageSize(Descriptors.FieldDescriptor fieldDescriptor, Mess
                 (message.getSerializedSize()) + message.getSerializedSize();
     }
 
+    private String anyMessageTypeNameFromTypeUrl(String typeUrl) {
```

Comment:
Isn't it better to move these to utils?

Reply:
Since this is only related to the Any type messages, I thought of keeping it here.